### ISSUE TITLE
diffInBusinessDays  and Carbon 3 returns negative without BusinessTime::RELATIVE_DIFF

### ISSUE THREAD

**Issue by makroxyz** [kylekatarnls/business-time#79]
Upgraded to Carbon 3.x 

All diffInBusiness* functions works as before except diffInBusinessDays which returns negative value also w/o RELATIVE_DIFF flag

```php
now()->diffInBusinessDays(Illuminate\Support\Carbon::parse('2024-12-01')) // --> -10
Illuminate\Support\Carbon::parse('2024-12-01')->diffInBusinessDays(now()) // -->10

now()->diffInBusinessHours(Illuminate\Support\Carbon::parse('2024-12-01')) // -->76.0
Illuminate\Support\Carbon::parse('2024-12-01')->diffInBusinessHours(now())  //--> 76.0

now()->diffInBusinessMinutes(Illuminate\Support\Carbon::parse('2024-12-01')) // --> 4560.0
Illuminate\Support\Carbon::parse('2024-12-01')->diffInBusinessMinutes(now()) // --> 4560.0

now()->diffInBusinessSeconds(Illuminate\Support\Carbon::parse('2024-12-01')) // -->  273600.0
Illuminate\Support\Carbon::parse('2024-12-01')->diffInBusinessSeconds(now()) // -->  273600.0
```

**Comment by kylekatarnls** [comment]
Hello, IMO this is consistent (avoid to have diffInBusiness* not aligned with diffIn* methods).

This allows you to know from it the direction of the difference and if you don't need it, `abs()` is a more explicit way to get the absolute difference.

**Comment by makroxyz** [comment]
> Hello, IMO this is consistent (avoid to have diffInBusiness* not aligne with diffIn* methods).
> 
> This allows you to know from it the direction of the difference and if you don't need it, `abs()` is a more explicit way to get the absolute difference.

Yes... but all methos diffIn* don't have the minus sign except this one (diffInBusinessDays)!
In Carbon 2.x they all have the same behavior

**Comment by kylekatarnls** [comment]
You're right, they should all be relative (i.e. negative when compared date is before current date). I will investigate.

**Comment by makroxyz** [comment]
Hi can you release a fix?

**Comment by kylekatarnls** [comment]
Behavior has been aligned in [2.0.0](https://github.com/kylekatarnls/business-time/releases/tag/2.0.0) and will work the same with Carbon 2 and 3:

All values will be positive (absolute diff) by default, then you can get relative values with the `BusinessTime::RELATIVE_DIFF` option:
```php
->diffInBusinessDays(now(), BusinessTime::RELATIVE_DIFF)
->diffInBusinessHours(now(), BusinessTime::RELATIVE_DIFF)
```
